services/learn-card-network/brain-service/src/routes/boosts.ts

```diff
@@ -287,6 +296,166 @@ export const boostsRouter = t.router({
             return true;
         }),
 
+    getBoostAdmins: profileRoute
+        .meta({
+            openapi: {
+                protect: true,
+                method: 'POST',
+                path: '/boost/admins/{uri}',
+                tags: ['Boosts'],
+                summary: 'Get boost admins',
+                description: 'This route returns the admins for a boost',
+            },
+        })
+        .input(
+            PaginationOptionsValidator.extend({
+                limit: PaginationOptionsValidator.shape.limit.default(25),
+                includeSelf: z.boolean().default(false),
```

Comment:
Does it make sense for this to be default false? As in, would it be confusing to not get yourself if you are specifically querying for boost admins? 

But maybe it does - thoughts?

Reply:
I think when I first thought about this I'd thought it would make sense to default to false, but actually now that you're bringing it up, I think it would make more sense to be true by default. I'll go ahead and update!